1-js/02-first-steps/07-operators/article.md

```diff
@@ -1,168 +1,169 @@
 
 
 ```js run
-alert(2 + 2 + '1' ); // "41" and not "221"
+alert(2 + 2 + '1' ); // "41", a nie "221"
 ```
 
-String concatenation and conversion is a special feature of the binary plus `+`. Other arithmetic operators work only with numbers and always convert their operands to numbers.
+Konkatenacja i konwersja ciągów znaków to specjalne własności dwuargumentowego plusa `+`. Pozostałe operatory arytmetyczne działają tylko na liczbach i zawsze konwertują swoje argumenty do typu liczbowego.
 
-For instance, subtraction and division:
+Przykład z odejmowaniem i dzieleniem:
 
 ```js run
 alert( 2 - '1' ); // 1
 alert( '6' / '2' ); // 3
 ```
 
-## Numeric conversion, unary +
+## Konwersja na liczbę, jednoargumentowy +
 
-The plus `+` exists in two forms: the binary form that we used above and the unary form.
+Operator plusa `+` ma dwa warianty: dwuargumentowy (użyty powyżej) oraz jednoargumentowy.
 
-The unary plus or, in other words, the plus operator `+` applied to a single value, doesn't do anything to numbers. But if the operand is not a number, the unary plus converts it into a number.
+Jednoargumentowy plus lub, po prostu, operator plusa `+` zastosowany na jednej wartości nie wpływa w żaden sposób na liczby. Jeśli jednak argument nie jest typu liczbowego, ten operator przekonwertuje go do liczby.
 
-For example:
+Na przykład:
 
 ```js run
-// No effect on numbers
+// Brak efektu na liczbach
 let x = 1;
 alert( +x ); // 1
 
 let y = -2;
 alert( +y ); // -2
 
 *!*
-// Converts non-numbers
+// Konwertuje nie-liczby
 alert( +true ); // 1
 alert( +"" );   // 0
 */!*
 ```
 
-It actually does the same thing as `Number(...)`, but is shorter.
+W zasadzie robi on to samo co `Number(...)`, ale ma krótszy zapis.
 
-The need to convert strings to numbers arises very often. For example, if we are getting values from HTML form fields, they are usually strings. What if we want to sum them?
+Bardzo często zdarza się, że musimy przekonwertować tekst na liczbę. Na przykład, jeśli pobierzemy wartość z pola formularza HTML, zazwyczaj będzie ona typu tekstowego. Co jeśli zechcemy dodać ją do wartości innego pola?
 
-The binary plus would add them as strings:
+Dwuargumentowy plus po prostu by je połączył:
 
 ```js run
 let apples = "2";
 let oranges = "3";
 
-alert( apples + oranges ); // "23", the binary plus concatenates strings
+alert( apples + oranges ); // "23", dwuargumentowy plus łączy napisy
 ```
 
-If we want to treat them as numbers, we need to convert and then sum them:
+Jeśli chcemy potraktować te wartości jako liczby, musimy je najpierw przekonwertować, a następnie zsumować:
 
 ```js run
 let apples = "2";
 let oranges = "3";
 
 *!*
-// both values converted to numbers before the binary plus
+// Obydwie wartości są konwertowane do liczb, a następnie przekazywane do dwuargumentowego plusa
```

Comment:
Why start with uppercase letter when similar comment on line 118 starts with lowercase and so does original version of line 114?